examples/viu/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM --platform=$BUILDPLATFORM tonistiigi/xx:rust AS build
+RUN apt-get install -y git
+WORKDIR /src
+RUN git clone git://github.com/wapm-packages/viu
+WORKDIR /src/viu
+ARG TARGETPLATFORM
+RUN cargo build --release --out-dir=/out
+
+FROM debian:buster-slim AS release-linux
+COPY --from=build /out/ /usr/bin/
+ENTRYPOINT ["viu"]
+
+FROM scratch AS release-wasi
+COPY --from=build /out/viu.wasm /
+ENTRYPOINT ["/viu.wasm"]
+
+FROM release-$TARGETOS AS release
+ADD https://media.giphy.com/media/9FvN85CcQU9fW/giphy.gif /success.gif
```

Comment:
if you put the images in the release image, just bring the `CMD ["/docker.png"]` here.

Reply:
ok `release` needs to work with stdin.